.changeset/witty-boats-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': minor
```

Comment:
Do we really need minor version bump? Isn't `patch` sufficient?

Reply:
that's the question indeed

Reply:
thanks, makes sense, I was not aware of `patch`.
Corrected to `patch` with commit 26c545bef8dbbf0a7449d80f247b4f092aa1fec7